Makefile

```
@@ -249,10 +294,6 @@ lint-actions:
 	@echo " - linting GitHub actions"
 	actionlint
 
-$(addprefix lint_,$(go_modules)): lint_%:
-	@echo " - linting: $(patsubst lint_%,%,$@)"
-	@pushd src/$(patsubst lint_%,%,$@) >/dev/null && golangci-lint run --config ${lint_config} ${OPTS} --timeout 5m
-
 lint-gorouterproxy:
 	@echo " - linting: gorouterproxy"
 	@pushd src/autoscaler/integration/gorouterproxy >/dev/null && golangci-lint run --config ${lint_config} $(OPTS) --timeout 5m
```

Do we need the `--timeout` here any longer?
```suggestion
	@pushd src/autoscaler/integration/gorouterproxy >/dev/null && golangci-lint run --config ${lint_config} $(OPTS)
```